ignite/cmd/cmd.go

```diff
@@ -82,18 +82,18 @@ func logLevel(cmd *cobra.Command) chain.LogLvl {
 	return chain.LogRegular
 }
 
-func printEvents(wg *sync.WaitGroup, bus events.Bus, s *clispinner.Spinner) {
+func printEvents(wg *sync.WaitGroup, bus *events.Bus, s *clispinner.Spinner) {
```

Comment:
Aren't we going to move this func to `cliui.printLoop`?

Reply:
I left it because I didn't refactor cmd directory at all except `network_chain_list` and `network_chain_init` as examples.